var/spack/repos/builtin/packages/libical/package.py

```diff
@@ -16,7 +16,10 @@ class Libical(CMakePackage):
 
     depends_on('cmake@3.11.0:', type='build')
     depends_on('perl', type='build')
+    depends_on('pkgconfig', type='build')
+    depends_on('glib')
```

Comment:
```suggestion
    depends_on('glib@2.32:')
```
https://github.com/libical/libical/blob/v3.0.8/CMakeLists.txt#L395